packages/salesforcedx-sobjects-faux-generator/test/integration/sobjectTransformer.test.ts

```diff
@@ -0,0 +1,277 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { AuthInfo } from '@salesforce/core';
+import { fail } from 'assert';
+import { expect } from 'chai';
+import { EventEmitter } from 'events';
+import * as fs from 'fs';
+import * as path from 'path';
+import { createSandbox } from 'sinon';
+import { SObjectTransformer } from '../../src';
+import {
+  ERROR_EVENT,
+  EXIT_EVENT,
+  FAILURE_CODE,
+  STDERR_EVENT,
+  STDOUT_EVENT,
+  SUCCESS_CODE
+} from '../../src/constants';
+import { nls } from '../../src/messages';
+import { MinObjectRetriever } from '../../src/retriever';
+import { CancellationTokenSource } from './integrationTestUtil';
+
+const PROJECT_NAME = `project_${new Date().getTime()}`;
+const CONNECTION_DATA = {
+  accessToken: '00Dxx000thisIsATestToken',
+  instanceUrl: 'https://na1.salesforce.com'
+};
+
+const env = createSandbox();
+
+// tslint:disable:no-unused-expression
+describe('Transform sobject definitions', () => {
+  let cancellationTokenSource: CancellationTokenSource;
+  let projectPath: string;
+  let emitter: EventEmitter;
+
+  before(async () => {
```

Comment:
I've seen certain issues in the past when mixing `before` and `beforeEach` blocks. Can we update this to just have one ?